src/transformers/models/blip_2/modeling_blip_2.py

```diff
@@ -1789,6 +1789,13 @@ def forward(
             language_model_outputs=outputs,
         )
 
+    def can_generate(self) -> bool:
+        """
+        Returns True. This model can `generate` and must therefore have this property set to True in order to be used
+        in the visual-question-answering pipeline.
+        """
+        return True
```

Comment:
I'm going to ask to remove this change, and I'll add a more robust version myself -- in essence, classes with a custom `generate` should obviously return `True` here, and the general function does not indeed capture this case! 🤗

Reply:
`can_generate()` was updated, this can be safely removed (it now returns `True` for the model, without the overload)

Reply:
Thanks a lot, custom can_generate was removed from [Blip2ForConditionalGeneration](https://github.com/huggingface/transformers/pull/25532/commits/d309aa4f913ee823f8f04a58177fc009d643e6af)